packages/2d/src/components/Latex.ts

```diff
@@ -29,6 +29,35 @@ export interface LatexProps extends ImgProps {
   renderProps?: SignalValue<OptionList>;
 }
 
+/**
+ * A node for rendering equations with LaTeX.
+ *
+ * @preview
+ * ```tsx editor
+ * import {Latex} from '@motion-canvas/2d/lib/components';
+ * import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+ * import {waitFor} from '@motion-canvas/core/lib/flow';
+ * import {createRef} from '@motion-canvas/core/lib/utils';
```

Comment:
Since `3.9.0` [all entities should be imported from the main entry of our packages](https://motioncanvas.io/blog/version-3.9.0#new-features)